transform_to_tf/include/transform_to_tf/transform_to_tf.hpp

```diff
@@ -0,0 +1,72 @@
+#ifndef TRANSFORM_TO_TF_H_
+#define TRANSFORM_TO_TF_H_
+
+#include <string>
+#include <vector>
+
+#include <ros/ros.h>
+#include <Eigen/Geometry>
+
+#include <eigen_conversions/eigen_msg.h>
+#include <minkindr_conversions/kindr_msg.h>
+#include <minkindr_conversions/kindr_tf.h>
+#include <kindr/minimal/quat-transformation.h>
+#include <tf/transform_broadcaster.h>
+
+namespace transform_to_tf {
+
+// Default values for parameters
+static const bool kDefaultVerbose = true;
+static const std::string kDefaultLocalFrameName = "body";
+static const std::string kDefaultGlobalFrameName = "world";
+
+// Convenience typedef
+typedef kindr::minimal::QuatTransformation Transformation;
+
+struct TransformationStamped {
+  ros::Time stamp;
+  Transformation transformation;
+};
+
+// Class handling global alignment calculation and publishing
+class TransformToTf {
+ public:
+  // Constructor
+  TransformToTf(const ros::NodeHandle& nh, const ros::NodeHandle& nh_private);
+
+ protected:
```

Comment:
In general I have been making things protected to give as much flexibility to inheriting classes. But yeah, I checked the style guide and says to make things private except where needed. Good call.